### PR TITLE
feat: Add enablePassThrough prop

### PR DESCRIPTION
**Pull request by varholak-peter**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -47,18 +55,18 @@ export default class Carousel extends React.Component {
...
     }, () => {
       this.scroller = new Scroller(
         scrollerElement,
-        Object.assign({}, {
-          scrollingY: vertical,
-          scrollingX: !vertical,
-          scrollBoundary: 70,
-          scrollResponseBoundary: 70,
-        }, scrollerOptions)
+        Object.assign({}, scrollingOptions, passThroughOptions, scrollerOptions)
```
Comment (umbobabo) [inline src/index.js]: scrollingOptions and scrollerOptions are really confusing names.

File: src/index.js [hunk 2]
```diff
@@ -167,6 +175,19 @@ export default class Carousel extends React.Component {
     event.preventDefault();
   }
 
+  createKeyDownHandle(clickHandle) {
+    return function handleKeyDown(evt) {
```
Comment (umbobabo) [inline src/index.js]: We usually don't use abbreviation...

File: src/index.js [hunk 3]
```diff
@@ -222,6 +244,7 @@ export default class Carousel extends React.Component {
             style={{ display: hideNextButton ? 'none' : '' }}
             direction="next"
             onClick={this.handleNextClick}
+            onKeyDown={this.createKeyDownHandle(this.handleNextClick)}
```
Comment (umbobabo) [inline src/index.js]: createKeyDownHandle ... probably should be createKeyDownHandle**r**

Comment (varholak-peter) [inline src/index.js]: renamed

File: src/index.js (hunk 2 above)
Comment (varholak-peter) [inline src/index.js]: renamed

File: src/index.js (hunk 1 above)
Comment (varholak-peter) [inline src/index.js]: renamed

**Pull request merged**